tests/test_exception_handling.py

```diff
@@ -64,7 +64,7 @@ def test_exception_handler_with_unknown_argument():
         (
             'problem_exists_exception',
             HTTPStatus.FORBIDDEN,
-            'application/json+problem',
+            'application/problem+json',
```

Comment:
do we want to use MediaType.APPLICATION_PROBLEM_JSON here?

Reply:
i would say no, because if we rename it after that these tests passed successfully and we will not catch the problem.